designsafe/static/scripts/data-depot/components/modals/nees-doi-list.component.js

```diff
@@ -0,0 +1,30 @@
+import NeesDoiListTemplate from './nees-doi-list.template.html';
+
+class NeesDoiListCtrl {
+    constructor () {
+    }
+
+    $onInit() {
+        this.project = this.resolve.project;
+        this.experimentDois = this.project.metadata.experiments.map((exp) => ({
+            title: exp.title,
+            DOI: exp.doi,
+            doiURL: `https://doi.org/${exp.doi}`
+        }));
+    }
+    
+    close() {
+        return;
+    }
+}
+
+export const NeesDoiListComponent = {
+    template: NeesDoiListTemplate,
+    controller: NeesDoiListCtrl,
+    controllerAs: '$ctrl',
+    bindings: {
+        resolve: '<',
+        close: '&',
+        dismiss: '&'
+    },
+};
```

Comment:
Also, if you catch an existing page or one you've created without a newline at the end go ahead an add one. I'm not sure why this is needed, but Sal had mentioned I should keep an eye out for this as well.